tests/pos/nullAsInstanceOf.scala

```diff
@@ -1,11 +0,0 @@
-object Test {
```

Comment:
That's a compatibility adaptor that works only if you do not inspect arguments. And that is the current situation with run tests.